ncl/ncl_entries/days_in_month.ipynb

```diff
@@ -0,0 +1,217 @@
+   "id": "f762cbca2cf59dfa",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-06-06T20:18:11.403172Z",
+     "start_time": "2024-06-06T20:18:11.400471Z"
+    }
+   },
+   "source": [
+    "import cftime\n",
+    "\n",
+    "day = 1\n",
+    "month = 2\n",
+    "year = 1500\n",
+    "\n",
+    "days = cftime.datetime(year, month, day, calendar='standard').daysinmonth\n",
+    "print(f\"standard calendar: \\t\\t\\t\\t month {month} in {year} has {days} days\")\n",
+    "\n",
+    "days = cftime.datetime(year, month, day, calendar='proleptic_gregorian').daysinmonth\n",
+    "print(f\"proleptic Gregorian calendar: \\t month {month} in {year} has {days} days\")\n",
+    "\n",
+    "days = cftime.datetime(year, month, day, calendar='julian').daysinmonth\n",
+    "print(f\"Julian calendar: \\t\\t\\t\\t month {month} in {year} has {days} days\")\n",
+    "\n",
+    "days = cftime.datetime(year, month, day, calendar='360_day').daysinmonth\n",
+    "print(f\"360 day calendar: \\t\\t\\t\\t month {month} in {year} has {days} days\")"
+   ],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "standard calendar: \t\t\t\t month 2 in 1500 has 29 days\n",
+      "proleptic Gregorian calendar: \t month 2 in 1500 has 28 days\n",
+      "Julian calendar: \t\t\t\t month 2 in 1500 has 29 days\n",
+      "360 day calendar: \t\t\t\t month 2 in 1500 has 30 days\n"
+     ]
+    }
+   ],
+   "execution_count": 3
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a8b079a3a3eb9f00",
+   "metadata": {},
+   "source": [
+    "## Differences between `cftime` and NCL's `days_in_month`\n",
+    "\n",
+    "### Calendars\n",
+    "The calendars available in `cftime` and NCL's `days_in_month`, while both based on the CF Conventions calendars, are slightly different.\n",
+    "\n",
+    "- `cftime` does support using `\"none\"` as a calendar type, but not for this particular function. \n",
+    "- NCL's `days_in_month` does not support the `proleptic_gregorian` calendar.\n",
+    "\n",
+    "### Numerical Differences\n",
+    "Additionally, the `\"standard\"` and `\"gregorian\"` calendars seem to disagree for some historical dates.\n",
+    "\n",
+    "### Input type\n",
+    "Notably, using `cftime` to calculate the number of days in a month works by getting a `daysinmonth` value from a `cftime.datetime` object, which means that the calculations for days in a month have to be collected through each date individually, while NCL's `days_in_month` can take in multidimensional integer arrays, given that the `year` and `month` arrays have the same dimensions."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8eda876fd519891",
+   "metadata": {},
+   "source": [
+    "## Python Resources\n",
+    "- The `calendar` module [documentation](https://docs.python.org/3/library/calendar.html).\n",
+    "- The [`cftime.datetime` documentation](https://unidata.github.io/cftime/api/cftime.datetime.html).\n",
```

Comment:
This link may be broken. I think this might be the page you were looking for.
```suggestion
    "- The [`cftime.datetime` documentation](https://unidata.github.io/cftime/api.html#cftime.datetime).\n",
```

Reply:
Good catch!